features.yaml

```diff
@@ -96,23 +98,23 @@ features:
     area: Traffic Management
     id: "traffic.k8s_gateway_apis"
   - name: "Kubernetes Gateway APIs for mesh (`Service` `parentRef`) "
+    id: "traffic.k8s_gateway_apis_+mesh"
     link: "/docs/tasks/traffic-management/"
     level:
       checklist: features/k8s-gateway-apis.md
-      maturity: Beta
+      maturity: Stable
```

Comment:
Remember having some subset not being stable, hope this was not intentional.

Reply:
[PTAL here](https://docs.google.com/document/d/14gJnJ3IsEzZS1VHmA3TvGh19P-gN1cFHMR9P47kEzSw/edit) if you care to: the direction is basically "GA things can have their non-GA things listed separately". 

Gateway API features that are Experimental are going to be "Alpha/Beta" etc in Istio, but I think it's fair to declare GAMMA is stable.

This PR is cleanup moving towards no longer listing *everything* on this page.  We may end up removing everything, listing only stable things, or using this as a source to update the docs pages to more correctly talk about things being Experimental/Alpha/Beta at the individual feature level.